et_ee.lang

```diff
@@ -415,3 +416,55 @@ gui.xaero_box_pushbox_potion_effects=Läbipaistev kast ümber vanilli võlujoogi
 gui.xaero_held_centered_position=Keskendamisel käes olevad esemed
 gui.xaero_held_centered_position_screen_edges=servades
 gui.xaero_held_centered_position_hotbar=plokiribal
+gui.xaero_minimap_frame=Raami stiil
+gui.xaero_minimap_frame_color=Raami värv
+gui.xaero_box_minimap_frame_color=Minikaardi raami värv mitte-vaikimisi värvitud stiilide korral.
+gui.xaero_minimap_frame_default=vaikimisi
+gui.xaero_minimap_frame_colored_thick=värvitud
+gui.xaero_minimap_frame_colored_thin=värvitud õhuke
+gui.xaero_compass_north=N
+gui.xaero_compass_east=E
+gui.xaero_compass_south=S
+gui.xaero_compass_west=W
```

Comment:
Are these untranslated on purpose?

Reply:
Yes, Estonian uses full words of 3-5 characters so I thought it looks better with English here, like a compass.

Reply:
Alright, great!